src/site/content/en/blog/declarative-link-capturing/index.md

```diff
@@ -26,9 +26,16 @@ origin_trial:
   url: https://developer.chrome.com/origintrials/#/view_trial/4285175045443026945
 ---
 
-{% Aside %} Declarative Link Capturing is part of the
-[capabilities project](https://web.dev/fugu-status/) and is currently in development. This post will
-be updated as the implementation progresses. {% endAside %}
+{% Aside 'warning' %} Declarative Link Capturing was part of the
+[capabilities project](https://web.dev/fugu-status/)
+and is no longer pursued. The feature will be reincarnated as `launch_handler` and `scope_extensions`.
+Watch this site for updates. {% endAside %}
+
+{% Details %}
+
+{% DetailsSummary %}
+Original article
+{% endDetailsSummary %}
```

Comment:
What is this?

Reply:
Just added you on the email thread.

Reply:
I saw that thanks. I had to read that email thread to understand the context of this. Web.dev readers won't have that context. In any case, shouldn't this say something like, "The features described in this article are no longer being pursued"?

Reply:
Please see the [staged preview](https://deploy-preview-5950--web-dev-staging.netlify.app/declarative-link-capturing/), it does exactly that on the top and in the title and leaves the original article accessible. (It’s the same for both articles.)

Reply:
So it makes the article content collapsable. To what end?

Reply:
Just the entire article, apart from the warning aside.